### PR TITLE
Update D:\Git\winget-pkgs\manifests\m\mIRC\mIRC\7.72

### PR DESCRIPTION
**Pull request by Trenly**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/114499&drop=dogfoodAlpha

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
> Installer Verification Analysis Context Information: 
Specified hash doesn't match.
Actual hash:   4358DF45209E9703EC378D41BE04792290E3D7AE7E1AE7F71C31C2A0681F19E3
Expected hash: b04292ae7356a133fd64b99807dd5352792f027db54209a79ef76f5415816421

File: manifests/m/mIRC/mIRC/7.72/mIRC.mIRC.installer.yaml [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
 MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
 InstallerSuccessCodes:
 - 1223
 Installers:
 - Architecture: x86
-  InstallerType: nullsoft
   InstallerUrl: https://www.mirc.com/downloads/mirc/39/mirc772.exe
-  InstallerSha256: 6BAF43FEF743770BBC17722E2D982088C020EB41EC52F6C6E6D1289E24D0A6BA
+  InstallerSha256: B04292AE7356A133FD64B99807DD5352792F027DB54209A79EF76F5415816421
```
Comment (stephengillie) [inline manifests/m/mIRC/mIRC/7.72/mIRC.mIRC.installer.yaml:13]:
```suggestion
  InstallerSha256: 4358DF45209E9703EC378D41BE04792290E3D7AE7E1AE7F71C31C2A0681F19E3
```

**Review by ItzLevvie — APPROVED** [review]

**Comment by Trenly** [comment]
image

**Review by ImJoakim — APPROVED** [review]

**Pull request merged**